src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -109,7 +109,7 @@ def config_dashboard(self, daemon_descrs: List[DaemonDescription]):
         """Config dashboard settings."""
         raise NotImplementedError()
 
-    def get_active_daemon(self, daemon_descrs: List[DaemonDescription]) -> DaemonDescription:
+    def get_active_daemon(self, daemon_descrs: List[DaemonDescription]) -> Optional[DaemonDescription]:
```

Comment:
Hm. What's the difference between an empty list and None here?

Reply:
@sebastian-philipp I'm a bit confused by what you mean here. This function doesn't return a list.

Reply:
whoops.